Cache/Main.lean

```diff
@@ -59,11 +64,22 @@ def toPaths (args : List String) : List FilePath :=
 
 /-- Commands which (potentially) call `curl` for downloading files -/
 def curlArgs : List String :=
-  ["get", "get!", "get-", "put", "put!", "put-unpacked", "commit", "commit!"]
+  ["get", "get!", "get-", "miniget", "miniget-", "put", "put!", "put-unpacked", "commit", "commit!"]
 
 /-- Commands which (potentially) call `leantar` for decompressing downloaded files -/
 def leanTarArgs : List String :=
-  ["get", "get!", "pack", "pack!", "unpack", "lookup"]
+  ["get", "get!", "miniget", "miniget-", "pack", "pack!", "unpack", "lookup"]
+
+-- Copied from ImportGraph/RequiredModules.lean.
+namespace Lean.NameSet
+
+def ofList (l : List Name) : NameSet :=
```

Comment:
This means we cannot import both files, right? That sounds bad...? Also, this one is unused.

Reply:
Yes, this means we cannot import both files. No, I don't think this is bad: I don't think `Cache/Main.lean` is ever *imported* (only run, when this doesn't matter). Finally: if leanprover-community/batteries#1091 gets merged, the code can be re-used from there...

Reply:
And that bump also got merged.